docs/user-guides/guardrails-library.md

```diff
@@ -772,6 +773,27 @@ rails:
 
 For more details, check out the [Private AI Integration](./community/privateai.md) page.
 
+
+### Prompt Security Protection
+
+NeMo Guardrails supports using [Prompt Security API](https://prompt.security/?utm_medium=github&utm_campaign=nemo-guardrails) for protecting input and output retrieval flows.
+
+To activate the protection, you need to set the `PS_PROTECT_URL` and `PS_APP_ID` environment variables.
```

Comment:
Here it'd be better to refer to this variables in prompt.security documentation. what do you think?